pkg/trtl/config/testdata/replicas.json

```diff
@@ -0,0 +1,34 @@
+{
+    "40": {
+        "enabled": true,
+        "pid": 40,
+        "region": "brooklyn",
+        "name": "raphael",
+        "gossip_interval": 1200000000000,
```

Comment:
Unfortunately Go `time.Duration` serializes as number of nanoseconds by default and it won't be fixed until Go 2.0 - https://github.com/golang/go/issues/10275

I could have written a custom type that deserialized a string such as "2m"  - but this type would not have played well easily with envconfig.